cmake/FindRust.cmake

```
@@ -187,8 +187,10 @@ function(_corrosion_determine_libs_new target_triple out_libs out_flags)
                 endif()
             endforeach()
             set(libs_list "${stripped_lib_list}")
-            # Special case `msvcrt` to link with the debug version in Debug mode.
-            list(TRANSFORM libs_list REPLACE "^msvcrt$" "\$<\$<CONFIG:Debug>:msvcrtd>")
+            # We leave it up to the C/C++ executable that links in the Rust static-library
+            # to determine which version of the msvc runtime library it should select.
+            list(FILTER libs_list EXCLUDE REGEX "^msvcrtd?")
+            list(FILTER flags_list EXCLUDE REGEX "^/defaultlib:msvcrtd?")
```

```suggestion
            list(FILTER flag_list EXCLUDE REGEX "^/defaultlib:msvcrtd?")
```

A minor typo, it seems. On my windows box I'm still getting `/defaultlib:msvcrt` on the link flag due to this:

```
-- Determining required link libraries for target x86_64-pc-windows-msvc
-- Required static libs for target x86_64-pc-windows-msvc: kernel32;advapi32;kernel32;ntdll;userenv;ws2_32;kernel32;ws2_32;kernel32
-- Required link flags for target x86_64-pc-windows-msvc: /defaultlib:msvcrt
```

After fixing the typo, it filters out as expected:

```
-- Required static libs for target x86_64-pc-windows-msvc: kernel32;advapi32;kernel32;ntdll;userenv;ws2_32;kernel32;ws2_32;kernel32
-- Required link flags for target x86_64-pc-windows-msvc:
```

And then the warning goes away:

```
-- Required static libs for target x86_64-pc-windows-msvc: kernel32;advapi32;kernel32;ntdll;userenv;ws2_32;kernel32;ws2_32;kernel32
-- Required link flags for target x86_64-pc-windows-msvc:
```